docs/docusaurus.config.js

```diff
@@ -32,10 +58,12 @@ const config = {
       /** @type {import('@docusaurus/preset-classic').Options} */
       ({
         docs: {
+          sidebarItemsGenerator: async () => {
```

Comment:
Leveraged this documented approach here: https://typedoc-plugin-markdown.org/plugins/docusaurus/sidebar#injecting-into-autogenerated-sidebar